src/bin/sleep.rs

```diff
@@ -3,19 +3,86 @@
 extern crate coreutils;
 
 use std::env;
+use std::io::{self, Write};
 use std::thread;
-use std::io;
+use std::time::Duration;
 
-use coreutils::extra::{OptionalExt, fail};
 
-#[allow(deprecated)]
+use coreutils::extra::{OptionalExt};
+
+static MAN_PAGE: &'static str = r#"NAME
+    sleep - delay for a specified amount of time
+
+SYNOPSIS
+    sleep NUMBER[SUFFIX]...
+    sleep OPTION
+
+DESCRIPTION
+    Pause the shell for NUMBER seconds. An optional SUFFIX may be applied, such as 's' for seconds (default), 'm' for minutes, 'h' for hours or 'd' for day. Given multiple arguments, it will pause for the amount of time specified by the sum of their values. This implementation supports floating point numbers as input.
+
+    Example: sleep 60; sleep 1m; sleep 30s 30s;
+             sleep 0.5; sleep 0.5s; sleep 0.25s 0.25s
+
+OPTIONS
+    -h, --help
+        display this help and exit
+
+AUTHOR
+    Written by Jeremy Soller and Michael Murphy.
+"#;
+
+static MISSING_OPERAND: &'static str       = "sleep: missing operand";
```

Comment:
Well, since the error _should_ be written to stderr, the shell can potentially normalize it by prefixing the command.